homeassistant/components/todoist/strings.json

```
@@ -1,4 +1,23 @@
 {
+  "config": {
+    "step": {
+      "user": {
+        "data": {
+          "token": "[%key:common::config_flow::data::api_token%]"
+        },
+        "description": "Please entry your API token from your [Todoist Settings page]({settings_url})"
```

I think you may want `enter` here.

```suggestion
        "description": "Please enter your API token from your [Todoist Settings page]({settings_url})"
```